plasmapy/utils/decorators/checks.py

```diff
@@ -397,29 +397,23 @@ class CheckUnits(CheckBase):
         import astropy.units as u
         from plasmapy.utils.decorators import CheckUnits
 
-        @CheckUnits(arg1={'units': u.cm},
-                    arg2=u.cm,
-                    checks_on_return=[u.cm, u.km])
+        @CheckUnits(arg1={"units": u.cm}, arg2=u.cm, checks_on_return=[u.cm, u.km])
         def foo(arg1, arg2):
             return arg1 + arg2
 
         # or on a method
         class Foo:
-            @CheckUnits(arg1={'units': u.cm},
-                        arg2=u.cm,
-                        checks_on_return=[u.cm, u.km])
+            @CheckUnits(arg1={"units": u.cm}, arg2=u.cm, checks_on_return=[u.cm, u.km])
             def bar(self, arg1, arg2):
                 return arg1 + arg2
 
     Define units with function annotations::
 
-        import astropy.units as u
-        from plasmapy.utils.decorators import CheckUnits
-
```

Comment:
I did end up removing the repeated import statements, since they were included in the first code block.